apps/full-site-editing/full-site-editing-plugin/donations/donations.js

```diff
@@ -6,53 +6,106 @@ import classNames from 'classnames';
 /**
  * WordPress dependencies
  */
-import { useState } from '@wordpress/element';
+import { InspectorControls, __experimentalBlock as Block } from '@wordpress/block-editor';
+import { Button, ExternalLink, PanelBody, ToggleControl } from '@wordpress/components';
+import { useEffect, useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
-import { Button } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import StripeNudge from './stripe-nudge';
 
-const Donations = ( { products, stripeConnectUrl } ) => {
+const Donations = ( { attributes, products, setAttributes, siteSlug, stripeConnectUrl } ) => {
+	const { oneTimePlanId, monthlyPlanId, annuallyPlanId, showCustomAmount } = attributes;
 	const [ activeTab, setActiveTab ] = useState( 'one-time' );
 
-	const isActive = ( button ) => ( activeTab === button ? 'active' : null );
+	const isTabActive = ( tab ) => activeTab === tab;
+
+	useEffect( () => {
+		if ( ! oneTimePlanId ) {
+			setAttributes( { oneTimePlanId: products[ 'one-time' ] } );
+		}
+	}, [ oneTimePlanId ] );
 
 	return (
-		<div className="donations__container">
-			{ stripeConnectUrl && <StripeNudge stripeConnectUrl={ stripeConnectUrl } /> }
-			<div className="donations__tabs">
-				<Button className={ isActive( 'one-time' ) } onClick={ () => setActiveTab( 'one-time' ) }>
-					{ __( 'One-Time', 'full-site-editing' ) }
-				</Button>
-				<Button className={ isActive( 'monthly' ) } onClick={ () => setActiveTab( 'monthly' ) }>
-					{ __( 'Monthly', 'full-site-editing' ) }
-				</Button>
-				<Button className={ isActive( 'annually' ) } onClick={ () => setActiveTab( 'annually' ) }>
-					{ __( 'Annually', 'full-site-editing' ) }
-				</Button>
-				<div
-					id="donations__tab-one-time"
-					className={ classNames( 'donations__tab', { active: isActive( 'one-time' ) } ) }
-				>
-					{ __( 'One time', 'full-site-editing' ) }
-				</div>
-				<div
-					id="donations__tab-monthly"
-					className={ classNames( 'donations__tab', { active: isActive( 'monthly' ) } ) }
-				>
-					{ __( 'Monthly', 'full-site-editing' ) }
-				</div>
-				<div
-					id="donations__tab-annually"
-					className={ classNames( 'donations__tab', { active: isActive( 'annually' ) } ) }
-				>
-					{ __( 'Annually', 'full-site-editing' ) }
+		<>
```

Comment:
We shouldn't need the fragment any more, now with the `<Block.div>`.

Reply:
Good eye @kwight, I've updated this in https://github.com/Automattic/wp-calypso/pull/43798/commits/4e11be462db68bea1c655ea0e5de0547653830b1